src/Aspire.Hosting/Publishing/DockerContainerRuntime.cs

```diff
@@ -9,11 +11,41 @@ namespace Aspire.Hosting.Publishing;
 internal sealed class DockerContainerRuntime(ILogger<DockerContainerRuntime> logger) : IContainerRuntime
 {
     public string Name => "Docker";
-    private async Task<int> RunDockerBuildAsync(string contextPath, string dockerfilePath, string imageName, CancellationToken cancellationToken)
+    private async Task<int> RunDockerBuildAsync(string contextPath, string dockerfilePath, string imageName, ContainerBuildOptions? options, CancellationToken cancellationToken)
     {
+        var arguments = $"build --file {dockerfilePath} --tag {imageName}";
```

Comment:
Use `buildx` here instead of `build`. Update the CheckIfRunningAsync method to check if the `docker buildx`command is available.